changelog_entry.yaml

```
@@ -0,0 +1,4 @@
+bump: patch
```

And indent remaining to fix the test
```suggestion
bump: - patch
```

---

@MaxGhenis,  Thanks for the help on the changelog_entry format!

---

@MaxGhenis,  Your suggested change to the `changelog_entry.yaml `file did not work.  So, I'm still puzzled about the error.

---

@MaxGhenis,  I finally corrected my mistake on the changelog_entry YAML format.